python/cudf/cudf/tests/test_reshape.py

```diff
@@ -71,7 +71,7 @@ def test_melt(nulls, num_id_vars, num_value_vars, num_rows, dtype):
 @pytest.mark.parametrize(
     "dtype",
     list(NUMERIC_TYPES + DATETIME_TYPES)
-    + [pytest.param("str", marks=pytest.mark.xfail())],
+    + [pytest.param("str", marks=pytest.mark.xfail())],  # type: ignore
```

Comment:
Do we need this anymore now that tests are ignored?

Reply:
Tests aren't being ignored currently -- should we?

Reply:
Is there any reason for tests to provide type annotations? (annotations for test utilities, I can see being helpful, but not limited-scope test code).